<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>980</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>980</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>980</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MiniBasic</string>
  </property>
  <widget class="QLabel" name="label_code">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>代码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_output">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>运行结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>语句与语法树</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_code">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>380</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_output">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>380</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_tree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>781</width>
     <height>411</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>900</y>
     <width>253</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>载入代码(LOAD)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_run">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>900</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>执行代码(RUN)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>538</x>
     <y>900</y>
     <width>253</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>清空代码(CLEAR)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_order">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>930</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>命令输入窗口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>950</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
